lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1121,12 +1121,17 @@ func (w *WalletKit) FundPsbt(_ context.Context,
 			}
 		}
 
+		lockDuration := time.Duration(0)
+		if req.LockDuration != 0 {
+			lockDuration = time.Duration(req.LockDuration)
```

Comment:
Need to multiply by `time.Second`, otherwise this will end up being nanoseconds.

Reply:
Just realized this after messing around with the type in the playground. Will get this fixed